src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -47,14 +47,22 @@ class ValidateActor(wdlSource: WdlSource, wdlJson: WdlJson)
 
   private def validateWorkflow(sentBy: ActorRef): Unit = {
     logger.info(s"$tag for $sentBy")
-    val futureValidation: Future[Unit] = for {
+    val futureValidation: Future[NamespaceWithWorkflow] = for {
       namespaceWithWorkflow <- Future(NamespaceWithWorkflow.load(wdlSource))
-      inputs <- Future(wdlJson.parseJson).map(_.asJsObject.fields)
-      coercedInputs <- Future.fromTry(namespaceWithWorkflow.coerceRawInputs(inputs))
       runtime = namespaceWithWorkflow.workflow.calls foreach { _.toRuntimeAttributes }
-    } yield () // Validate that the future run and return `Success[Unit]` aka (), or `Failure[Exception]`
 
-    futureValidation onComplete {
+    } yield namespaceWithWorkflow // Validate that the future run and return `Success[Unit]` aka (), or `Failure[Exception]`
+
+    val futureValidationIncludingInputs: Future[Unit] = workflowInputs match {
+      case None => futureValidation map { x => }
```

Comment:
Didn't realize that would compile. :smile: I would have done `x => ()` but (shrug)